src/CMakeLists.txt

```diff
@@ -27,40 +49,105 @@ if(WIN32)
 
     set(RADARE2_INCLUDE_DIRS "${IAITO_WIN32_DIR}/radare2/include/libr" "${IAITO_WIN32_DIR}/include")
 endif()
+#
+# Win32 setting ends
 
-
-find_package(Radare2 REQUIRED)
-include_directories(${RADARE2_INCLUDE_DIRS})
-link_directories(${RADARE2_LIBRARY_DIRS})
-
-
-if(CMAKE_CXX_COMPILER_ID STREQUAL "GNU"
-        OR CMAKE_CXX_COMPILER_ID STREQUAL "Clang")
-    add_definitions(-Wall -Wextra)
+# Apple
+if(${CMAKE_SYSTEM_NAME} MATCHES "Darwin")
+	set(CMAKE_CXX_FLAGS "-mmacosx-version-min=10.7 -std=gnu0x -stdlib=libc++")
 endif()
 
 
-# Parse iaito.pro to get filenames
-include(QMakeProParse)
-parse_qmake_pro("${CMAKE_CURRENT_SOURCE_DIR}/iaito.pro" IAITO_PRO)
-set(SOURCE_FILES ${IAITO_PRO_SOURCES})
-set(HEADER_FILES ${IAITO_PRO_HEADERS})
-set(UI_FILES ${IAITO_PRO_FORMS})
-set(QRC_FILES ${IAITO_PRO_RESOURCES})
-
-message(STATUS "sources from iaito.pro: ${SOURCE_FILES}")
-message(STATUS "headers from iaito.pro: ${HEADER_FILES}")
-message(STATUS "forms from iaito.pro: ${UI_FILES}")
-message(STATUS "resources from iaito.pro: ${QRC_FILES}")
-
+# Compiler settings
+#
+if(CMAKE_CXX_COMPILER_IS_GNUCXX OR CMAKE_CXX_COMPILER_IS_CLANGXX)
+	set(CMAKE_CXX_FLAGS "-std=c++11 -Wall -Wextra")
+endif(CMAKE_CXX_COMPILER_IS_GNUCXX OR CMAKE_CXX_COMPILER_IS_CLANGXX)
```

Comment:
Please never set compiler flags directly if not really necessary. For C++11, only `set(CMAKE_CXX_STANDARD 11)` should be used. Or do you have a specific reason to not use it?

And if you really want to set CMAKE_CXX_FLAGS manually, only append it and do not overwrite it completely unless you explicitly want to do that.

Also, `CMAKE_CXX_COMPILER_IS_GNUCXX` and `CMAKE_CXX_COMPILER_IS_CLANGXX` do not exist. I think it's best to check `CMAKE_CXX_COMPILER_ID` as it was before.